qemu/tests/qemu_disk_img_convert.py

```diff
@@ -33,6 +36,28 @@ def convert(self, t_params=None):
         self.trash.append(converted)
         return params
 
+    @error_context.context_aware
+    def compare_test(self, t_params):
+        """
+        Compare images.
+
+        :param t_params: Dictionary with the test parameters
+        """
+        error_context.context("Compare images with default and strict mode",
+                              logging.info)
+        for compare_mode in t_params.objects("compare_mode_list"):
+            if compare_mode == "strict":
+                t_params["compare_mode"] = compare_mode
+            image1 = self.image_filename
+            image2 = storage.get_image_filename(t_params, self.data_dir)
+            try:
+                super(ConvertTest, self).compare_images(t_params, image1, image2)
+                if compare_mode == "strict":
+                    raise error.TestFail("images are identical in strict mode")
+            except (exceptions.TestFail, exceptions.TestError), detail:
+                if compare_mode != "strict":
+                    raise error.TestFail("images are mismatch: %s" % detail)
+
```

Comment:
hi, the line 77 should be deleted "n_params = convert_test.create_snapshot()" , didn't see the steps in polarion case.